.github/workflows/sync_pytorch_canary.yml

```diff
@@ -0,0 +1,29 @@
+name: sync_pytorch_canary
+
+on:
+  schedule:
+    # Every 15 minutes
+    - cron: "*/15 * * * *"
```

Comment:
I wanted to do something more unique, like "03/15 * * * *", but that's nonstandard and I don't know if GHA will handle it properly.